app/src/main/java/com/example/parkingsystem_kotlin/util/ExtensionFunctions.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.parkingsystem_kotlin.util
+
+import android.content.Context
+import android.widget.Toast
+
+fun Context.toast(message: String) =
+    Toast.makeText(this, message, Toast.LENGTH_SHORT).show()
```

Comment:
this can fi on a single line

Reply:
The Android Studio is a very strict Indentator :p 
![image](https://user-images.githubusercontent.com/565557/127546250-111998d5-3b08-4aa0-895a-a0a061427607.png)